### PR TITLE
Add ctrl+s save to edit form

### PR DESCRIPTION
**Pull request by negreirosleo**
Pressing Ctrl+S will try to save the current active form.

[Screencast from 2023-12-21 12-20-47.webm](https://github.com/Igalia/phpreport/assets/20492786/9027eb98-451c-438f-b8bf-886d07402816)

### CODE REVIEW

**Review by dmtrek14 — APPROVED** [review]
LGTM - thanks!

File: frontend/src/app/tasks/hooks/useEditTaskForm.ts [hunk 1]
```diff
@@ -51,5 +53,19 @@ export const useEditTaskForm = ({ task, tasks, closeForm }: UseEditTaskFormProps
...
+    const handleKeyPress = (event: KeyboardEvent) => {
+      if (event.key === 's' && event.ctrlKey && formRef.current?.contains(document.activeElement)) {
+        event.preventDefault()
+        formRef.current?.requestSubmit()
+      }
+    }
+    document.addEventListener('keydown', handleKeyPress)
+
+    return () => {
+      document.removeEventListener('keydown', handleKeyPress)
```
Comment (anarute) [inline frontend/src/app/tasks/hooks/useEditTaskForm.ts:66]: Why do we need these 2 keydown events?

Comment (negreirosleo) [reply frontend/src/app/tasks/hooks/useEditTaskForm.ts:66]: It's actually just one, the second part is removing the eventListener when the component unmounts

Comment (anarute) [reply frontend/src/app/tasks/hooks/useEditTaskForm.ts:66]: Oh, got it now, sorry

**Review by anarute — APPROVED** [review]

**Pull request merged**